cmake/win32/CreateFixupBundle.cmake

```diff
@@ -0,0 +1,32 @@
+#
+function(create_fixup_bundle)
+    set(options)
+    set(oneValueArgs BINARY_PATH SCRIPT_OUTPUT_PATH)
+    set(multiValueArgs LIBRARIES_PATHS IGNORE_ITEMS)
+    cmake_parse_arguments(BUNDLE "${options}" "${oneValueArgs}"
+            "${multiValueArgs}" ${ARGN} )
+
+      configure_file(
+        "${LMMS_SOURCE_DIR}/cmake/win32/FixupBundle.gen.cmake"
```

Comment:
We usually use `.in` extension for files to be configured. So, I'm suggesting `FixupBundle.cmake.in` for the name of the input file and `FixupBundle.cmake` for the output.